Cargo.toml

```diff
@@ -46,40 +46,41 @@ predicates = "1.0.5"
 prettytable-rs = "0.8.0"
 rand = "0.7.3"
 regex = "1.4.1"
-reqwest = { version = "0.10.9", features = ["blocking", "json"] }
-rustls = "0.18.1"
+reqwest = { version = "0.11.3", features = ["blocking", "json", "multipart"] }
+rustls = "0.19.1"
 semver = "0.11.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.60"
 serde_with = "1.5.1"
 tempfile = "3.1.0"
 term_size = "0.3"
 text_io = "0.1.8"
-tokio = { version = "0.2", default-features = false, features = ["io-std", "time", "macros", "process", "signal", "sync"] }
-tokio-native-tls = "0.1.0"
-tokio-rustls = "0.14.1"
-tokio-tungstenite = { version = "0.11.0", features = ["tls"] }
+tokio = { version = "1.5.0", default-features = false, features = ["io-std", "time", "macros", "process", "signal", "sync"] }
+tokio-native-tls = "0.3.0"
+tokio-rustls = "0.22.0"
+tokio-stream = "0.1.5"
+tokio-tungstenite = "0.14.0"
 toml = "0.5.8"
 toml_edit = "0.2.0"
 twox-hash = "1.6.0"
 url = "2.2.0"
 uuid = { version = "0.8", features = ["v4"] }
 which = "4.0.2"
 ws = "0.9.1"
-
```

Comment:
Nit: Can you put this with the rest of the tokio deps? i think they're all alphabetized rn it would be nice to maintain that 🥺

Reply:
omg its not showing up but i meant to highlight tokio-retry

Reply:
You got it